index.html

```diff
@@ -22,17 +22,329 @@
+        <h1>Iptisam Souhel</h1>
+    </div>
+    <div class="container3">
+        <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+    If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+    </div>
+    <div class="container4">
+        <div class="info">
+            <p>Start collaboration</p>
         </div>
-        <div class="container3">
-            <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
-         If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
-        </div>
-        <div class="container4">
-            <div class="info">
-                <p>Start collaboration</p>
-            </div>
+    </div>
+    
+    <nav class="container5">
+        <ul class="images">
+            <li class="shape3"><img src="img/Shape1.png" alt="shape"></li>
+            <li class="shape4"><img src="img/Shape2.png" alt="shape"></li>
+            <li class="shape5"><img src="img/Shape3.png" alt="shape"></li>
+            <li class="shape6"><img src="img/Shape4.png" alt="shape"></li>
+            <li class="shape7"><img src="img/Shape5.png" alt="shape"></li>
+            <li class="shape8"><img src="img/Shape7.png" alt="shape"></li>
+            <li class="shape9"><img src="img/Shape8.png" alt="shape"></li>
+            <li class="shape10"><img src="img/Shape9.png" alt="shape"></li>
+            <li class="shape11"><img src="img/Shape10.png" alt="shape"></li>
+            <li class="shape12"><img src="img/Shape11.png" alt="shape"></li>
+            <li class="shape13"><img src="img/Shape12.png" alt="shape"></li>
+            <li class="shape14"><img src="img/Shape13.png" alt="shape"></li>
+            <li class="shape15"><img src="img/Shape14.png" alt="shape"></li>
+            <li class="shape16"><img src="img/Shape15.png" alt="shape"></li>
+            <li class="shape17"><img src="img/Shape.png" alt="shape"></li>
+            
+        </ul>
+    </nav>
+    <div class="container6">
+        <h2>Projects</h2>
+    </div>
+    
+    <nav class="container7">
+        <ul class="comp1">
+            <li class="rect1"><img src="imgformobile/Rectangle1.png" alt="computer"></li>
+        </ul>
+    </nav>
+    <nav class="container8">
+        <ul>
+            <li class="Proj1"><a href="#">Hello-World</a></li>
+        </ul>
+        <nav class="container9">
+            <ul class="data1">
+                <li class="Lang"><a href="#">HTML/CSS</a></li>
+            </ul>
+        </nav>
+        <nav class="container10">
+            <ul class="data2">
+                <li class="Lang1"><a href="#">Ruby on </a></li>
+            </ul>
+        </nav>
+        <nav class="container11">
+            <ul class="data3">
+                <li class="Lang2"><a href="#">JavaScript</a></li>
+            </ul>
+        </nav>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`, please remove all nav tags 👍🏿